util/cron/common-cray-cs.bash

```diff
@@ -27,7 +27,7 @@ else
 fi
 
 # Point clang to standard libraries
-export COMPILER_PATH=$GCC_X86_64
+export COMPILER_PATH="$(dirname $(dirname $(g++ --print-file-name=libstdc++.so)))"
```

Comment:
Can we stuff this in a helper/common script with a comment about why/when it's needed. Something like `common.llvm-comp-path.bash`